modules/preview-react/form-field/lib/FormFieldLabel.tsx

```diff
@@ -43,10 +43,16 @@ export const formFieldLabelStencil = createStencil({
     },
 
     // orientation modifier from parent FormField
-    [parentModifier(formFieldStencil.modifiers.orientation.horizontal)]: {
+    [parentModifier(formFieldStencil.modifiers.orientation.horizontalStart)]: {
       float: 'left',
       maxHeight: system.space.x10,
     },
+    // orientation modifier from parent FormField
+    [parentModifier(formFieldStencil.modifiers.orientation.horizontalEnd)]: {
+      float: 'left',
```

Comment:
I don't think we need `float` here as the parent container is a `flex` element and it has a `flex-direction: 'row'`. 
```suggestion
```

Reply:
we need float when we use the label in the context of a fieldset and legend :(

Reply:
@NicholasBoll